examples/Console/UseServicePlanIdForSms.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Sinch;
+using Sinch.SMS;
+using Sinch.SMS.Batches.Send;
+
+namespace Examples
+{
+    public class UseServicePlanIdForSms
+    {
+        public void Example()
+        {
+            var sinchClient = new SinchClient(default, default, default, options =>
+            {
+                options.UseServicePlanIdWithSms(Environment.GetEnvironmentVariable("SINCH_SERVICE_PLAN_ID"),
```

Comment:
It looks like the servicePlanId is a second class citizen for SMS: the user still have to go to the SinchClient initialization with the 3 parameters and then define a 4th parameter to provide the login informations

Reply:
Yes, it is, I'm keeping it that way.